src/lib.rs

```diff
@@ -15,6 +15,7 @@
 // Coding conventions
 #![recursion_limit = "256"]
 #![deny(dead_code, /* missing_docs, */ warnings)]
+#![allow(clippy::init_numbered_fields)]
```

Comment:
Probably this will go away after resolving https://github.com/LNP-BP/client_side_validation/issues/54

Reply:
Can you pls check that we still need this with the latest 0.59 stable release?

Reply:
Removing this allow statement presents this warning/error on 1.59:

![Screenshot from 2022-03-07 05-09-12](https://user-images.githubusercontent.com/285690/157032034-7a22f71d-c5fc-46db-b480-be82ad20f8d8.png)